src/editUser/editUser.controller.ts

```diff
@@ -0,0 +1,132 @@
+import { FileInterceptor } from "@nestjs/platform-express";
+import { Request, Response } from "express";
+import * as multer from "multer";
+import { v4 as uuidv4 } from "uuid";
+import * as sharp from "sharp";
+import {
+  Body,
+  Controller,
+  Get,
+  HttpStatus,
+  Patch,
+  Req,
+  Res,
+  UploadedFile,
+  UseGuards,
+  UseInterceptors,
+} from "@nestjs/common";
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
+
+import { EditUserService } from "./editUser.service";
+import { JwtAuthGuard } from "src/auth/guards/jwt-auth.guards";
+import { EditUserDto } from "./dto/edit-user.dto";
+
+const bucketName = process.env.BUCKET_NAME;
```

Comment:
Put all the process.env.xxx in a file named "config.ts".  And read the env variable in config.ts.